packages/dataviews/src/view-table.js

```diff
@@ -42,51 +42,43 @@ const sortingItemsInfo = {
 };
 const sortIcons = { asc: chevronUp, desc: chevronDown };
 
+const sanitizeOperators = ( field ) => {
+	let operators = field.filterBy?.operators;
+	if ( ! operators || ! Array.isArray( operators ) ) {
+		operators = [ OPERATOR_IN, OPERATOR_NOT_IN ];
+	}
+	return operators.filter( ( operator ) =>
+		[ OPERATOR_IN, OPERATOR_NOT_IN ].includes( operator )
+	);
+};
+
 function HeaderMenu( { field, view, onChangeView } ) {
-	const isSortable = field.enableSorting !== false;
 	const isHidable = field.enableHiding !== false;
+
+	const isSortable = field.enableSorting !== false;
 	const isSorted = view.sort?.field === field.id;
-	let filter, filterInView;
-	const otherFilters = [];
-	if ( field.type === ENUMERATION_TYPE ) {
-		let columnOperators = field.filterBy?.operators;
-		if ( ! columnOperators || ! Array.isArray( columnOperators ) ) {
-			columnOperators = [ OPERATOR_IN, OPERATOR_NOT_IN ];
-		}
-		const operators = columnOperators.filter( ( operator ) =>
-			[ OPERATOR_IN, OPERATOR_NOT_IN ].includes( operator )
+
```

Comment:
This was rewritten to make it clear and follow the naming schema of filters (`filter`, `filterInView`, `activeElement`, and `activeOperation`).